src/main/azure-functions-language-worker-protobuf/src/proto/FunctionRpc.proto

```diff
@@ -313,6 +376,34 @@ message TypedData {
   }
 }
 
+// Specify which type of data is contained in the shared memory region being read
+enum RpcDataType {
+  unknown = 0;
+  string = 1;
+  json = 2;
+  bytes = 3;
+  stream = 4;
+  http = 5;
+  int_x = 6;
```

Comment:
Why do we add x as suffix here?

Reply:
Updated name to int_x and double_x since int and double are keywords in java and generated grpc code throws error if key words are used as variable names. 
Since GRPC relies on the number for comparing, it is better to rename the variable to anything but int and double here.